backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -55,7 +55,7 @@ object Mailer {
                         .from(fromName, smtpConfig.username)
                         .withSubject(subject)
                         .withPlainText(message)
-                        .withHeader("Content-Type", "text/plain; charset=utf8")
+                        .withHeader("Content-Type", "text/plain; charset=UTF-8")
```

Comment:
Good catch. Maybe we also need to wrap it in quotes? Most mails I receive personally have it wrapped... (not sure what the standard says though) 🤔 , like so:

```suggestion
                        .withHeader("Content-Type", "text/plain; charset=\"UTF-8\"")
```

Reply:
I was also thinking about putting it into quotes, as e.g. google does it for mails they send, but the I kept it as it is, as the [RFC2045](https://www.ietf.org/rfc/rfc2045.txt) from 1992 already defined:

> the  following two forms
  Content-type: text/plain; charset=us-ascii (Plain text)
  Content-type: text/plain; charset="us-ascii"
  are completely equivalent.

So we should be fine with sticking to it without quote.